.github/workflows/sbt-dependency-graph.yml

```diff
@@ -15,5 +15,5 @@ jobs:
       - uses: actions/checkout@v3
       - uses: scalacenter/sbt-dependency-graph-action@v1
         with:
-          scala-versions: 2.11.12 2.12.16 2.13.8 3.1.3
+          scala-versions: 2.13.8
```

Comment:
Are you sure switching to only 2.13.8 will fix the issue? Looks like the issue was actually related to 2.13.

```
[error] sbt.librarymanagement.ResolveException: Error downloading org.scalameta:mtags_2.13.8:0.0.0+1-98d9d6b6-SNAPSHOT```

Reply:
ahhh, gotcha. 👍🏼